prometheus-to-sd/Makefile

```diff
@@ -18,7 +18,7 @@ ENVVAR = GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 IMAGE_NAME=prometheus-to-sd
 ALL_ARCH=amd64 arm64
 PREFIX ?= staging-k8s.gcr.io
-TAG ?= v0.11.9
+TAG ?= v0.11.10
```

Comment:
can you bump this to 0.11.11? A 0.11.10 tag already exists https://github.com/GoogleCloudPlatform/k8s-stackdriver/commits/prom-to-sd-v0.11.10

Reply:
Done